MIGRATION.md

```diff
@@ -0,0 +1,7 @@
+# Migration Guide
+
+## Migration to Version 7
```

Comment:
Let's refer to the tag (`v7.0.0`)

Reply:
For a migration guide this documentation should cover migration to any Version 7 version, not just 7.0.0. I could see using semantic versioning, i.e., "Migration to Version ^7.0.0". Would that work?